### PR TITLE
Fetch exomol

### PR DESCRIPTION
**Pull request by minouHub**
### Description

This pull request is to address issue #585.

Inspecting the file, I found several problems, mostly relating to the calculation of the broadening parameter (gamma and N) in the [compute_broadening](https://github.com/radis/radis/blob/0b87b6f9e0541fce9f6c9e8b7be30ffefbdd3c30/radis/api/exomolapi.py#L1326) function of exomol.api:

- [x] First of all, the name of the functions is confusing. The actual pressure broadening FWHM is not calculated here. Only the broadening coefficients are calculated. 
- [x] The handling of the exception is not well organized. A very large `try` `except` is employed. This should be avoided as much as possible.
- [ ] It is not possible to ask for another broadener than "air". 
- [x] `verbose=0` has no impact on the print. 
- [x] in the specific case of CO2, the above error is raised because the Dataframe is in `vaex` format (instead of Pandas historically)

### CODE REVIEW

File: radis/lbl/loader.py [hunk 1]
```diff
@@ -931,6 +931,7 @@ def fetch_databank(
         load_columns="equilibrium",
         parallel=True,
         extra_params=None,
+        verbose=True,
```
Comment (erwanp) [inline radis/lbl/loader.py:934]: There is a SpectrumFactory.verbose parameter, no need to create a new parameter in the function

Just use 
```
if self.verbose
```
or 
```
verbose = self.verbose
if verbose ... 
```

(also, not related but always good to mention, we can have multiple verbose levels like 
```
if verbose >= 2: print("some extra details")
```

**Review by erwanp — CHANGES_REQUESTED** [review]
(see my comment on using self.verbose rather than verbose)

File: radis/lbl/loader.py (hunk 1 above)
Comment (minouHub) [reply radis/lbl/loader.py]: self.verbose ==> indeed, done

**Comment by erwanp** [comment]
Failing with https://app.travis-ci.com/github/radis/radis/jobs/607057429#L2301  : remaining `sf.fetch_databank(**conditions, verbose=verbose)`   in `_calc_spectrum_one_molecule()`   @minouHub

**Comment by minouHub** [comment]
The last feature (broadening with other molecules than air) is not critical and will be sorted in #602

**Review by erwanp — APPROVED** [review]

**Pull request merged**